domain.yml

```diff
@@ -1,753 +1,864 @@
-  utter_direct_to_step2:
-    - text: "You can click the button below to go to the next step and to try out Rasa once you're ready."
-      buttons:
-        - title: "Continue"
-          payload: "/get_started_step2"
-  utter_continue_step2:
-    - text: "Let's continue, click the button below"
-      buttons:
-        - title: "Continue"
-          payload: "/get_started_step2"
-  utter_continue_step4:
-    - text: "Let's continue, click the button below"
-      buttons:
-        - title: "Continue"
-          payload: "/get_started_step4"
-  utter_no_more_steps:
-    - text: "Looks like you've already completed all the steps!"
-  utter_explain_job_function:
-    - text: "Your job function helps us in understanding how we can talk to you best."
-  utter_explain_use_case:
-    - text: "In order to help you best, we need to know what you want to do."
-  utter_explain_budget:
-    - text: "If you tell us your budget, we can prepare a package that fits you best."
-  utter_explain_person_name:
-    - text: "To be able to contact you later on, we would like to know your name."
-  utter_explain_company:
-    - text: "Just for completeness."
-  utter_explain_business_email:
-    - text: "We need your business email to contact you afterwards. Otherwise, we cannot help you set up Rasa."
+  - text: 'Here a few reasons why you should contribute:
+
+      - Explore the source code in depth
+
+      - Check out issues and help solve them
 
+      - Contribute to open source software
+
+      '
+  utter_recommend_forum:
+  - text: I suggest signing up for our [forum](https://forum.rasa.com) to stay up to
+      date on product updates, events, and resources.
+  utter_response_why_email:
+  - text: I need your email address to sign you up to the newsletter.
+  - text: I need it so that I can sign you up to the newsletter.
+  utter_restart_with_button:
+  - buttons:
+    - payload: /restart
+      title: Restart
+    text: Click the button below if you want to start over.
+  utter_run_rasa_init:
+  - text: You can create a new Rasa project by running 'rasa init'. This creates the
+      necessary training and configuration files, and also trains your first model using
+      some example training data.
+  utter_sales_contact:
+  - text: Thanks, now I need to know how we can reach you.
+  utter_salesrequest_failed:
+  - text: "Unfortunately we weren't able to store your information. Sorry about that!\
+      \ You can still contact sales via our [website](https://rasa.com/contact-sales/)\
+      \ 🙂"
+  utter_search_bar:
+  - text: Try using the search bar above for now.
+  utter_source_code:
+  - text: You can find my source code [here](https://github.com/RasaHQ/rasa-demo/).
+  utter_spacy:
+  - text: 'SpaCy has excellent pre-trained named-entity recognizers for this -- check
+      out this [interactive demo](https://demos.explosion.ai/displacy-ent/). I would
+      suggest using the SpacyEntityExtractor component in your pipeline.
+
+      '
```

Comment:
I think we can move some of these strange trailing quotes up to the end of the statement (e.g. right after pipeline) -- not sure though